tests/assets/function.yaml

```diff
@@ -11,5 +11,7 @@ command: gensky -c -B '{{ inputs.parameters.desired-irradiance }}' > '{{ inputs.
 outputs:
   artifacts:
     - name: sky
+      location: project-folder
       path: '{{ inputs.parameters.sky-file }}'
```

Comment:
So are we removing the quotes or not?

Reply:
Missed that one. I would double quote it :smile: